docs/reference/commandline/network_inspect.md

```diff
@@ -30,6 +30,9 @@ The `network inspect` command shows the containers, by id, in its
 results. For networks backed by multi-host network driver, such as Overlay,
 this command also shows the container endpoints in other hosts in the
 cluster. These endpoints are represented as "ep-{endpoint-id}" in the output.
+However, for Swarm mode services, only the endpoints that are local to the node
```

Comment:
I think "swarm scoped networks" was the general term we use, but I defer to @mavenugo